docs/ChangeLog

```diff
@@ -1,5 +1,38 @@
 # -*- mode: sh; fill-column: 120; -*-
 
+Version 3.0 (15 September 2023)
+
+[Important Highlights/Notices for 3.x series]
+
+  * the 3.x series targets support for three new major OS distro versions: EL 9, openSUSE Leap 15.5 and openEuler 22.03.
+     o As the OpenHPC 3.x series targets major new distro versions, please note that the 3.x series
+       is **not** backwards compatible with the OpenHPC 2.x series.
+     o OpenHPC 3.x is intended for fresh installs with the newer distro versions (or containers based
+       on these distro versions).
+     o Note that 3.x builds for SUSE based systems are now performed against Leap instead of SLES. Leap 15 is
+       intended to be compatible with SLES 15.
+     o EL 9 based builds are performed against RHEL 9.2 and tested against Rocky Linux 9.2 and AlmaLinux 9.2
```

Comment:
```suggestion
     o EL 9 based builds are performed against RHEL 9.2 and tested against Rocky Linux 9.2 and AlmaLinux 9.2
     o OpenHPC 3.x is the first release that supports openEuler OS! It is tested against openEuler 22.03 LTS
```

Reply:
Here I try to emphasize that OpenHPC 3.0 is the **first** release with support for openEuler.
Also I think we should make it somehow known that there are known issues with 22.03 LTS SP1 and SP2 (related to the ABI breaks in libicu)